pyproject.toml

```diff
@@ -24,7 +24,7 @@ version = "0.13.0"
 APScheduler = ">=3.11.0"
 aiohttp = ">=3.11.9"
 python = ">=3.9"
-redis = ">=5.2.0"
+redis = {version=">=5.2.0", optional=true}
```

Comment:
The CI failed to run the unit testing because of this changes. As it says in the log, please update the lock file and make sure all the tests pass.